handlers.c

```diff
@@ -432,11 +452,13 @@ bool handler__list(globals_t *vars, char **argv, unsigned argc)
     matches_and_old_values_swath *reading_swath_index = vars->matches->swaths;
     size_t reading_iterator = 0;
 
+    pager = get_pager();
+
     /* list all known matches */
     while (reading_swath_index->first_byte_in_child) {
         if (num == max_to_print) {
             if (num < vars->num_matches)
-                show_user("[...]\n");
+                fprintf(pager, "[...]\n");
```

Comment:
Here the `show_user()` was used because I don't want to print the `[...]` in backend mode.

Check manually for `options.backend` here or something equivalent.